src/app/_components/ProjectMenu.tsx

```diff
@@ -0,0 +1,88 @@
+"use client";
+
+import React, { useState } from "react";
+import { Button } from "t/components/ui/button";
+import { Card, CardContent, CardHeader, CardTitle } from "t/components/ui/card";
+import { cn } from "lib/utils";
+
+const ProjectMenu = () => {
```

Comment:
use function definition, not anonymous function for component.

```suggestion
export default function ProjectMenu() {
```